client/src/services/tournament.js

```diff
@@ -49,6 +49,23 @@ class Tournament {
 			})
 	}
 
+	teamsMatches(teamNumber){
+		if(this._teamsMatches && this._teamNumber && this._teamNumber == teamNumber){
+			return Promise.resolve(this._teamsMatches)
+		}
+
+		let self = this
+
+		this._teamNumber = teamNumber
+
+		return this.init()
+			.then(() => self.$http.get(`${self.tournament}/teams/${teamNumber}/matches`))
+			.then(response => {
+				self._teamsMatches = response.data

```

Comment:
If you cache it here like the other requests, you have to cache it per `teamNumber`, because different teams may have different matches